rb/Gemfile.lock

```diff
@@ -25,7 +25,6 @@ GEM
     irb (1.6.4)
       reline (>= 0.3.0)
     json (2.6.3)
-    json (2.6.3-java)
```

Comment:
Please undo this - I'm pretty sure it's necessary either by JRuby or TruffleRuby.